posthog/tasks/test/test_usage_report.py

```diff
@@ -56,6 +56,81 @@
 logger = structlog.get_logger(__name__)
 
 
+def _setup_replay_data(team_id: int, include_mobile_replay: bool) -> None:
+    # recordings in period  - 5 sessions
```

Comment:
we were looping here to add "multiple snapshots" but we don't store snapshots in replay table any more so we can avoid the inner loop and make this a _little_ cheaper to run at least 😊